rag_experiment_accelerator/doc_loader/tests/test_pdf_loader.py

```diff
@@ -0,0 +1,33 @@
+from unittest.mock import Mock
+
+import pytest
+from rag_experiment_accelerator.doc_loader.pdfLoader import load_pdf_files
+
+
+@pytest.mark.parametrize(
+    "pypdf_enabled, expected_content",
+    [
+        (True, "this is a sample pdf with text."),
+        (
+            False,
+            "this is a sample of text captured as an image and stored as pdf.this is a sample pdf with text.",
+        ),
```

Comment:
Severity: Critical - The `expected_content` for the `False` case seems to concatenate two sentences without any space or punctuation. Ensure this is the intended format.


[Hack - peer-review-pal] We are sorry for the intrusion. Feel free to ignore/delete the comments with the tag - they are automatically generated by [our Hack project](https://github.com/microsoft/peer-review-pal).
If you find them useful, want us to delete them or have any feedback or want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding #prp-ignore to your PR description.